### DISCUSSION TITLE
Create a option/icon for uploading a classified image in web platform using leafmap and geemap

### DISCUSSION THREAD

**Discussion by soumen-singhadeo** [opengeos/leafmap#381] [Q&A] [answered]
I want to develop a web platform and there create a icon/option for uploading a classified tif file. How can i create the upload option using leafmap and geemap in python ? After uploading the tif image i want to calculate landscape metrics from the uploaded tif file.
how can i create the upload option and calculate the landscape metrics?

**Comment by giswqs** [comment, marked as answer]
See

- https://geemap.org/notebooks/42_upload_data/
- https://youtu.be/4-WeaiObj84
